### PR TITLE
New Player Guide

### PR DESCRIPTION
**Pull request by Turtlelord26**
From Issue #207. Removed favored class text and added some of a character creation walkthrough aimed at new players.

Looking for a go-ahead from both technical and creative because I am trying to accomplish a creative goal (new-player-friendly) with a technical document.

### CODE REVIEW

File: CharacterCreation/01_Races.txt [hunk 1]
```diff
@@ -158,11 +154,31 @@ have this augment installed.
...
+causing the oceans to begin evaporating. With their oceans slowly dissipating 
+through the leak in the atmosphere, they have begun migrating to the nearby 
+Trioch IV, also mostly oceanus, which is out of immediate danger from the star. 
+Now, they live in both the oceans and the well-shaded estuaries on Trioch IV. A 
+few stubborn stragglers remain on Trioch II, which is slowly becoming a sandy, dry 
+wasteland with very little atmosphere. 
+
+
+Trioch II is now a place for venturing archaeologists to come and investigate the 
+way the Opaleites used to live.
```
Comment (bleehu) [inline CharacterCreation/01_Races.txt:181]: Thank you for justifying this! +1

File: CharacterCreation/00_New_Player_Walkthrough.txt [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+## Step 2: Class
+For players new to role-playing games we recommend the Soldier or Hitman classes (and possibly Pilot).
+*Section to be expanded later on account of impending large-scale class rework*
+
+## Step 3: Background and Skills
+*Section to be added on account of impending addition of this feature*
+
+## Step 4: Attribute Point Allocation
+Now that you know what your character is capable of in-combat (your class) and out-of-combat (your non-combat skills),
```
Comment (bleehu) [inline CharacterCreation/00_New_Player_Walkthrough.txt]: I know it's an intentional and helpful oversimplification, but I don't like saying that your class is for in-combat and you skills are for out-of-combat.

File: CharacterCreation/00_New_Player_Walkthrough.txt [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+Now that you know what your character is capable of in-combat (your class) and out-of-combat (your non-combat skills), 
+you have all the information you need to decide your character's attributes. You have a total number of attribute points
+to allocate based on your choice of race, and 7 attributes to which to allocate them. Basic Rules.txt describes what each 
+attribute does in a vacuum if you need a refresher. Here are a few pointers:
+1. Unless you know what you are doing, do not give your character less than 6 FORT. FORT directly determines your health
+points, and having enough of them contributes to not dying.
+2. Have at least 6 in either STR or DEX, and if possible up to 8. One of these stats will determine your movement speed,
+which is of high value.
+3. Have at least 6 Luck. Luck is a unique attribute and its value can be hard to gauge without experiencing its effects 
+firsthand.
```
Comment (bleehu) [inline CharacterCreation/00_New_Player_Walkthrough.txt]: I would suggest having at least 4 luck. I wouldn't say that your first character would need to be 'lucky'.

**Review by bleehu — CHANGES_REQUESTED** [review]
Most of these changes are non-blocking nitpicks, but before we merge, I would like to insist that we not merge with dev notes of future work. Either we write up how we want Races/Classes/Backgrounds to work and hold merging this from dev to RC, or preferably we yank the notes and add them as the features are ready.

File: CharacterCreation/00_New_Player_Walkthrough.txt (hunk 3 above)
Comment (Turtlelord26) [reply CharacterCreation/00_New_Player_Walkthrough.txt]: Fair enough. I can settle for just advising them to not be actively unlucky.

File: CharacterCreation/00_New_Player_Walkthrough.txt (hunk 2 above)
Comment (Turtlelord26) [reply CharacterCreation/00_New_Player_Walkthrough.txt]: Found a way to restate.

File: CharacterCreation/00_New_Player_Walkthrough.txt (hunk 3 above)
Comment (bleehu) [reply CharacterCreation/00_New_Player_Walkthrough.txt]: super nit-pick, the space is missing between "attribute pointes_to allocoate"

**Review by bleehu — APPROVED** [review]

**Review by bleehu — APPROVED** [review]
Make Shipit! Well done, Turtle!

**Comment by Turtlelord26** [comment]
@Reveraine This seems like a space you should live in so I would like your go-ahead too.
In particular, someone less technical should look at the document for clarity.

File: CharacterCreation/00_New_Player_Walkthrough.txt [hunk 4]
```diff
@@ -31,17 +31,17 @@ on what the attributes themselves do and the dependencies of the skills you sele
...
-##Step 6: Weapons, Attachments, and Armor
+## Step 6: Weapons, Attachments, and Armor
 Weapon and Armor proficiencies are determined by your class, as is your financial reach. Usually, though not always, you
 will want to acquire the heaviest type of armor your class has access to. Weapons are trickier, as each type of weapon
 is the best class of weapon for a particular task. As a [Class recommended in step 2], you may want a [X weapon for Y task],
 or a [Z weapon for W task]. [Sidearm discussion (if money permits)].
 
-##Step 7: Equipment
+## Step 7: Equipment
 Any remaining moneys you have can be spent on miscellaneous equipment from the [Items](https://github.com/bleehu/Compound_X/blob/play/CharacterCreation/07_Items.txt) document. Take whatever items you
```
Comment (Reveraine) [inline CharacterCreation/00_New_Player_Walkthrough.txt]: This should probably say 'credits' instead of money, but that's not a big change.

**Review by Reveraine — CHANGES_REQUESTED** [review]

**Review by Reveraine — APPROVED** [review]

File: CharacterCreation/00_New_Player_Walkthrough.txt (hunk 4 above)
Comment (bleehu) [reply CharacterCreation/00_New_Player_Walkthrough.txt]: Good Aesthetic Catch!

**Pull request merged**